lib/airbrake/rake/tasks.rb

```diff
@@ -100,7 +100,10 @@
 
     exit!(1) if [id, key].any?(&:nil?)
 
+    env = heroku_env['RAILS_ENV'] || ENV['RAILS_ENV']
```

Comment:
I think the fallback `ENV['RAILS_ENV']` in most cases will be `development`, since that's where this rake task will normally be run from. Maybe simply `heroku` is a less confusing default, but I may be missing something?